helm/helm.go

```diff
@@ -67,6 +67,11 @@ const SystemNamespace = "kube-system"
 
 const versionAll = "all"
 
+// the label Helm places on Kubernetes objects for differentiating between
+// different instances: https://helm.sh/docs/chart_best_practices/#standard-labels
+const HelmReleaseNameLabelLegacy = "release"
```

Comment:
Can we move this to `pkg/helm` instead? I'd very much like to get rid of this package.